### PR TITLE
USB: CaptureEye/OV511p webcam emulation

### PR DESCRIPTION
**Pull request by Florin9doi**
### Description of Changes

- CaptureEye/OV511p webcam emulation: #4763

image

### Suggested Testing Steps

- https://wiki.pcsx2.net/Police_24/7
- Verify that EyeToy games aren't impacted

### CODE REVIEW

**Comment by gilderoylockhart01** [comment]
Better than the gameboy camera ^^.

File: pcsx2/USB/usb-eyetoy/cam-windows.cpp [hunk 1]
```diff
@@ -230,12 +230,15 @@ namespace usb_eyetoy
...
 								{
-
 									VIDEOINFOHEADER* pVih = (VIDEOINFOHEADER*)pmtConfig->pbFormat;
 									pVih->bmiHeader.biWidth = frame_width;
 									pVih->bmiHeader.biHeight = frame_height;
 									pVih->bmiHeader.biSizeImage = DIBSIZE(pVih->bmiHeader);
 									hr = pSourceConfig->SetFormat(pmtConfig);
+									if (FAILED(hr))
+									{
+										Console.Warning("Camera: SetFormat err : %x", hr);
```
Comment (RedDevilus) [inline pcsx2/USB/usb-eyetoy/cam-windows.cpp:240]: Might be nice addendum to mention that the category is `USB:`

Comment (Florin9doi) [reply pcsx2/USB/usb-eyetoy/cam-windows.cpp:240]: "USB:" isn't used anywhere in this file

**Review by Mrlinkwii — APPROVED** [review]
`Police 24/7` works great with the  PR everything seems to be working as intended , checked some of the eyetoy games i own and saw no regressions :smile:  , everything worked out of the box

**Comment by PalFanTw** [comment]
The USB camera is not being detected when running the Japanese version of Shinjuku 24-shi

**Pull request merged**